### PR TITLE
added timeline stats

### PR DESCRIPTION
**Pull request by Yaroshvitaliy**

### CODE REVIEW

File: src/NBomber/DomainServices/TestHost/TestHostInfra.fs [hunk 1]
```diff
@@ -71,17 +70,19 @@ module internal TestHostReporting =
...
-                    |> Task.map(saveRealtimeStats dep.ReportingSinks)
-                    |> ignore
+                     let nodeStats = getNodeStats(operationTime)
+
+                     if not (List.isEmpty dep.ReportingSinks) then
+                        nodeStats
+                        |> Task.map(saveRealtimeStats dep.ReportingSinks)
+                        |> ignore
+
+                     timerTick(operationTime, nodeStats)
```
Comment (AntyaDev) [inline src/NBomber/DomainServices/TestHost/TestHostInfra.fs:80]: rename to addTimeLineStats

File: src/NBomber/DomainServices/Reporting/ViewModels.fs [hunk 2]
```diff
@@ -49,3 +55,13 @@ module NodeStatsViewModel =
 
     let serializeJson (viewModel: NodeStatsViewModel) =
         JsonConvert.SerializeObject(viewModel, Formatting.Indented)
+
+module TimeLineStatsViewModel =
+
+    let create (timeLineStats: (TimeSpan * NodeStats) list) = {
+        TimeStamps = timeLineStats |> List.toArray |> Array.map(fun (timeSpan, _) -> TimeSpan(0, 0, (int)timeSpan.TotalSeconds).ToString())
```
Comment (AntyaDev) [inline src/NBomber/DomainServices/Reporting/ViewModels.fs:62]: for such cases better use Seq.map |> Seq.toArray

File: src/NBomber/DomainServices/TestHost/TestHost.fs [hunk 3]
```diff
@@ -108,6 +106,16 @@ type internal TestHost(dep: GlobalDependency, registeredScenarios: Scenario list
         scnsProgressSubscriptions |> List.iter(fun x -> x.Dispose())
     }
 
+    let addTimeLineStats (timeLineStats: (TimeSpan * NodeStats) list)
+                         (operationTime: TimeSpan, nodeStats: Task<NodeStats list>) =
+        nodeStats
+        |> Task.map(fun nodeStatsList ->
```
Comment (AntyaDev) [inline src/NBomber/DomainServices/TestHost/TestHost.fs:112]: why Task here. NodeStats are available already

Comment (Yaroshvitaliy) [reply src/NBomber/DomainServices/TestHost/TestHost.fs:112]: getNodeStats returns Task:
getNodeStats: TimeSpan -> Task<NodeStats list>

**Comment by AntyaDev** [comment]
@Yaroshvitaliy This branch has conflicts that must be resolved

**Comment by Yaroshvitaliy** [comment]
> @Yaroshvitaliy This branch has conflicts that must be resolved

Done

**Pull request merged**